gateway/src/resty/openssl/x509.lua

```diff
@@ -55,7 +55,8 @@ local function normalize_pem_cert(str)
   if not str then return end
   if #(str) == 0 then return end
 
-  return re_gsub(str, [[\s(?!CERTIFICATE)]], '\n', 'oj')
+  -- using also jit compiler (j) will result in a segfault with some certificates
```

Comment:
Do you know which certificates cause the problem?

Reply:
The updated ones in this PR: t/fixtures/CA.  But only when they come from nginx, couldn't reproduce it in busted.